packages/frontend/src/notebook/notebook_cell.tsx

```diff
@@ -63,9 +62,6 @@ export type CellActions = {
     /** Delete this cell in the forward/downward direction. */
     deleteForward: () => void;
 
-    /** Duplicate this cell, adding the new cell below this one. */
-    duplicate: () => void;
-
```

Comment:
I don't think we should just remove the duplicate feature entirely. Instead:

- The `duplicate` field should become optional:

```typescript
duplicate?: () => void
```

- The menu item should be visible only when `duplicate` is defined
- It should be defined as before for all cell types *except* rich text